circle.yml

```diff
@@ -3,6 +3,7 @@ machine:
     GOPATH: "$HOME/go"
     IMPORT_PATH: "github.com/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME"
     COVERALLS_TOKEN: 69lPIvkLNM8MEDmGT8pBfJ0FECfSAvbjT
+    CODECOV_TOKEN: dcedb2f9-c70a-4883-8d8b-e8abf76d12b6
```

Comment:
We should keep these in circle under the sensitive env variables section so they can not be copied by everyone.